spec/rubocop/cop/rspec/spec_file_path_format_spec.rb

```diff
@@ -281,4 +281,14 @@ class Foo
       RUBY
     end
   end
+
+  context 'when configured with `IgnoreMetadata: { "foo" => ["bar"] }`' do
```

Comment:
I think maybe you should write a spec for `IgnoreMetadata: { "foo" => ["bar", "baz"] }` and validate that it ignores specs with either value. And that it does not ignore a spec without another value for `foo`, or no `foo` metadata at all.

Reply:
Fair, are the amended specs what you had in mind? I needed to play around with the wording a bit, since I was over the line length limit by one and two characters respectively, by using the existing phrasing.